news/2022-12-15-project-loved-december-2022.md

```diff
@@ -0,0 +1,372 @@
+- On the other hand, say hello to [Maxus](https://osu.ppy.sh/users/4335785)! He will be joining the metadata team to make sure that picks are in pristine condition before receiving their well-earned leaderboards.
+- A new drummer approaches! [Eriha](https://osu.ppy.sh/users/16320311) joins the osu!taiko Loved captains in order to help with bringing some of the community's most adored beatmaps to justice by granting them all a chance in the Loved polls.
+
+That should be all in terms of updates for this months. Don't forget if you're interested in participating in Project Loved discussions of a certain game mode, or have any questions, that you can reach out in one of the Project Loved Discord servers listed below:
+
+[![osu! Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-osu.png)](https://discord.gg/gn58Uk5sTE)
+
+[![osu!taiko Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-taiko.png)](https://discord.gg/BuJ6TVPmtN)
+
+[![osu!catch Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-catch.png)](https://discord.gg/phgtyS4UCh)
+
+[![osu!mania Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-mania.png)](https://discord.gg/Ededv7m)
+
+### Navigation
+
+- [osu!](#osu!)
+- [osu!taiko](#osu!taiko)
+- [osu!catch](#osu!catch)
+- [osu!mania](#osu!mania)
+
+## osu!
+
+The osu! Loved candidates were chosen by [DeviousPanda](https://osu.ppy.sh/users/4966334), [Mismagius](https://osu.ppy.sh/users/19048), [rHO](https://osu.ppy.sh/users/1629553), [Skill](https://osu.ppy.sh/users/7802027) and [Wixonater](https://osu.ppy.sh/users/10083084)!
+
+---
+
+[![](/wiki/shared/news/2022-12-15-project-loved-december-2022/1360992.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[wa. - Black Lotus](https://osu.ppy.sh/beatmapsets/1360992#osu) by [blixys](https://osu.ppy.sh/users/13015896)\
+200 BPM, 1:47 | 8.37★\
+*description written by [rHO](https://osu.ppy.sh/users/1629553)*
+
+If you don't know who [blixys](https://osu.ppy.sh/users/13015896) is, now is the time to meet them and their beautifully chaotic maps.
+
+Speaking in a language that mostly consists of heavily constructed rhythms, map structure and sliders, blixys shows us what they are capable of — capturing destruction and rebirth in their map *Black Lotus*. Using their mapping and object manipulation skills to the max, blixys successfully pulls off an jaw-dropping beatmap full of flowers and lotuses. I can hear the people saying *"Beatmaps are never that serious and/or poetic, rHO!"* comments already, but if you don't believe me, take a gander and you'll see what I'm talking about 😛
```

Comment:
Sorry, not sorry for the oxford comma
``` Suggestion
Speaking in a language that mostly consists of heavily constructed rhythms, map structure, and sliders, blixys shows us what they are capable of — capturing destruction and rebirth in their map *Black Lotus*. Using their mapping and object manipulation skills to the max, blixys successfully pulls off a jaw-dropping beatmap full of flowers and lotuses. I can hear the people saying *"Beatmaps are never that serious and/or poetic, rHO!"* comments already, but if you don't believe me, take a gander and you'll see what I'm talking about 😛
```

Reply:
news posts have more relaxed writing rules, including the oxford comma being optional

Reply:
still would be good to fix an -> a but yeah